pkgs/servers/mobilizon/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, mixRelease
+, callPackage
+, fetchMixDeps
+, fetchFromGitLab
+, git
+, cmake
+, nixosTests
+}:
+
+let
+  cldrLocales = callPackage ./cldrLocales.nix { };
+  js = callPackage ./js.nix { };
+in
+mixRelease rec {
+  pname = "mobilizon";
+  version = "1.1.0";
+
+  src = fetchFromGitLab {
+    domain = "framagit.org";
+    owner = "framasoft";
+    repo = pname;
+    rev = version;
+    sha256 = "1cmqpakjlafsnbk6c1vb6xzi5d2gwfihqxavbznjljk88wr2p4hr";
+  };
+
+  nativeBuildInputs = [ git cmake ];
+
+  mixDeps = fetchMixDeps {
+    inherit src pname version;
+    sha256 = "0cxpibgm9lfymicprpxa3d5i32lpwkgg107913bvyg53lajfbwpc";
+  };
+
+  preBuild = ''
+    # Else, the ex_cldr package will try to download locales at build time
+    cp -a ${cldrLocales}/* $MIX_DEPS_PATH/ex_cldr/priv/cldr/locales/
+
+    # Install the compiled js part
+    cp -a "${js}/libexec/mobilizon/deps/priv/static" ./priv
```

Comment:
Just noticed, it seems you're not running mix phx.digest anywhere.
It's not mandatory of course, but usually the release complains if it can't find the digest.

Reply:
What kind of errer would one have if the digests are missing? I'm not seeing any error mentioning digests

Reply:
The message shows on startup or on accessing a resource first if I remember correctly.
I'm about to have a go at packing another application, I'll post the warning message soon.

Reply:
I checked again the message, and I don't think it applies in the case of mobilizon.

Here is the error message
```
Could not find static manifest at \"/nix/store/njcsj769nkdrydipx1pzj67nazbx0xc8-bonfire-1.0.0/lib/bonfire-0.1.0-alpha.35/priv/static/cache_manifest.json\". Run \"mix phx.digest\" after building your static files or remove the configuration from \"config/prod.exs\"."}
```

I thought it was mandatory to define the cache_manifest location, but it's not. As long as it is not defined in the conf, it's not needed. I remember to have read somewhere that they recommend the digest to know when the content of thee file changes and to invalidate the cache, but again, it doesn't seem mandatory at all.

Reply:
Thanks for the update! I definitely don't remember seeing this message, despite having grepped for digest/manifest. After a little bit more investigation, I can see that the manifest does exist in the release, and that resources does contain the digest in the filename. I'm not sure how it got there, it seems to be done already by the yarn part